src/platforms/javascript/common/configuration/micro-frontend-support/multiplexed-transport.mdx

```diff
@@ -1,14 +1,13 @@
 ---
-title: Micro Frontend Support
-sidebar_order: 200
-description: ""
-keywords: ["micro frontend", "multiplexed transport"]
+title: Multiplexed Transport
+sidebar_order: 30
+description: "Send events to different Sentry projects based on the attributes on an event."
 ---
 
-The recommended way to use the Sentry SDK with Micro Frontend is to set up a multiplexed transport when calling `Sentry.init`. The multiplexed transport will send events to different Sentry projects based on the attributes on that event, which is how you can isolate events from a specific micro-frontend to a specific Sentry project.
-
 Requires SDK version `7.50.0` or higher.
 
+The multiplexed transport can send events to different Sentry projects based on the attributes on an event, which is how you can isolate events from a specific micro-frontend to a specific Sentry project.
```

Comment:
This intro sentence is much better at telling the user what they're getting out of the tool!